html-syntax/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Syntax</title>
+  </head>
+  <body>
+    <img src="http://www.smashbros.com/images/og/mewtwo.jpg" alt="Mewtwo" height="400" width="400" >
+    <p>Mewtwo</p>
+    <p>&copy Pokemon. &#174 Nintendo</p>
```

Comment:
Make sure you close your HTML entities with a `;`

Reply:
![Screen Shot 2021-10-04 at 9 02 12 PM](https://user-images.githubusercontent.com/72685552/135958224-80e104d1-36ea-45da-91f7-a6ac15524e67.png)

thanks for this!